website_multi_theme/README.rst

```diff
@@ -0,0 +1,159 @@
+
+Of course, your Odoo instance must be reachable by all of the provided host
+names, or nobody will ever see the effect. But that is most likely configured
+through your DNS provider and/or proxy, so it is not a matter of this addon.
+
+If you want to test this behavior, think that ``localhost`` and ``127.0.0.1``
+are different host names.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Follow the configuration steps.
+#. Enter any of the websites you modified.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/10.0
+
+Development FAQ
+===============
+
+How to develop a multiwebsite-ready theme?
+------------------------------------------
+
+Check ``demo/themes.xml``. It includes a demo theme that will serve as a
+template for you. This demo theme turns primary buttons green, so you can test
+if it is applied or not easily.
+
+How to convert a single-website theme in a multi-website one?
+-------------------------------------------------------------
+
+Check ``data/themes_bootswatch.xml``. You must do that. You can consider adding
+the support directly in this addon, given it will just do nothing if the
+single-website theme addon is not installed (it acts as a soft dependency).
+
+How to get multiwebsite-specific views updated?
+-----------------------------------------------
+
+This addon is conservative by default, meaning that in production databases
+views will not be updated if they already were created (except for the ones
+copied from ``templates/patterns.xml``).
+
+To force your website getting updated views for all views from a base theme
+that has changed, you should disable the website multi theme (to make the
+engine remove all views) and then re-enable it again (to recreate them from
+scratch).
+
+This does not happen in demo or development instances, where views arch is
+always updated.
+
+Known issues / Roadmap
+======================
+
+* Private themes support is not guaranteed.
+* There is no UI to remove websites. Do it through an odoo shell.
+* The module includes workarounds that should be removed once merged upstream:
+  * https://github.com/odoo/odoo/pull/17635
+  * https://github.com/odoo/odoo/pull/17970
+* Theme picker should include some kind of thumbnail if possible.
+* If you install any of the supported themes after installing this addon, you
+  will have to press *Reload* in the website config wizard to make it notice
+  the change.
+* If you install any unsupported theme along with this addon, it would possibly
+  become the base for all those supported, which can easily lead to weird
+  situations and errors.
+
+Required bug patch
+------------------
+
+You need to include this code in your Odoo instance for the addon to work.
+No workaround possible for this one:
+
+* https://github.com/odoo/odoo/pull/18462
```

Comment:
Indeed, it should be removed after applying https://github.com/OCA/website/pull/354/commits/0f2d8cf69a13d303d1416d31058d76edc1bc6401